app/models/models.scala

```diff
@@ -84,6 +84,13 @@ case class SSA (
   app: Option[String] = None
 ) {
   def isEmpty = stack.isEmpty && stage.isEmpty && app.isEmpty
+  def riffRaffLink: Option[String] = for {
+    stackName <- stack
+    stageName <- stage
+    appName <- app
+  } yield {
+    s"https://riffraff.gutools.co.uk/deployment/target/deploy?region=eu-west-1&stack=$stackName&stage=$stageName&app=$appName"
```

Comment:
Why is the region hard-coded?

Reply:
I also feel like the inclusion of this function on the model is borderline (leaning towards it being wrong).

Models are your datastructures. The rule of thumbs I use for including things on the models are:

* generally don't
* if it is an inherent property of the data structure and it entirely emerges from the information that the model contains then that probably makes sense to be included
* sometimes you really have to (e.g. `toString`)

In particular this means it mustn't take any arguments. While this function doesn't take any args and thus looks ok on the face of it, that's only because the Riff Raff URL is hard coded. That information isn't wholly contained in the data structure so I'd consider having this be a function that takes an SSA.

Reply:
The above is only a mater of taste though, so feel free to ignore me / come chat :-)

Reply:
p.s. the page this links to is very useful!